spring-grpc-docs/pom.xml

```diff
@@ -19,7 +19,6 @@
 		<configprops.path>${project.basedir}/src/main/antora/modules/ROOT/partials/_configprops.adoc</configprops.path>
 		<configprops.inclusionPattern>spring.grpc.*</configprops.inclusionPattern>
 		<jruby.version>9.4.6.0</jruby.version>
-		<disable.checkstyle.checks>true</disable.checkstyle.checks>
```

Comment:
It seems to be this that causes the build failure. I don't see why this should be a blocker.

Reply:
I think we can get this in after the release.